src/utils/rate-limit.js

```diff
@@ -25,27 +25,30 @@
  * @returns {function()}
  */
 export function rateLimit(win, callback, minInterval) {
-  let locker = null;
+  let locker = 0;
   let nextCallArgs = null;
 
-  const fire = args => {
-    callback.apply(null, args);
-
+  function fire(args) {
```

Comment:
why named function is better? both way create one function instance per `rateLimit()` call, right?